.commitlintrc.ts

```diff
@@ -0,0 +1,10 @@
+import type { UserConfig } from "@commitlint/types";
+
+const Configuration: UserConfig = {
+  extends: ["@commitlint/config-conventional"],
+  formatter: "@commitlint/format",
+  parserPreset: "conventional-changelog-conventionalcommits",
```

Comment:
```suggestion
```

The `formatter` is the default one ([see here](https://github.com/conventional-changelog/commitlint/blob/adc2e47bcfcd416361aa1381d54fee87153dbc6e/%40commitlint/load/src/load.ts#L83)).
The `parserPreset` is already configured by the extended config ([see here](https://github.com/conventional-changelog/commitlint/blob/adc2e47bcfcd416361aa1381d54fee87153dbc6e/%40commitlint/config-conventional/src/index.ts#L8)).

So we can safely delete those lines.